benchmarks/other_benchmarks/raw_sql/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+from ...utils import bench_setup
+
+bench_setup()
+
+
+class OneField(models.Model):
```

Comment:
Do we still need `OneFileld` in `benchmarks/models.py` once this is moved here?

Reply:
I have removed it